### DISCUSSION TITLE
Understanding and Resolving Issues with Image Dimensions in Percentage in MJML Editor

### DISCUSSION THREAD

**Discussion by Nishanth-sebastin** [mjmlio/mjml#2923] [Q&A]
1. When I set the width of an image using a percentage (e.g., 50%), it gets rendered as a pixel value (e.g., 50px). Why does this happen, and how can it be resolved?

2. When I set the height of an image using a percentage, the applied style does not reflect any visual change in the editor. Why does this occur, and how can it be addressed effectively?

**Comment by Nishanth-sebastin** [comment]
@iRyusa, if you have any insights or clarifications on this matter, your input would be greatly appreciated and highly beneficial to us.

**Comment by iRyusa** [comment]
Forgot to reply here, sorry I don't have notif on discussion. Outlook doesn't resolve image in % so we must use a pixel based width.

If you're using height in `px` this should be applied properly
